src/ir_LG.h

```diff
@@ -83,6 +84,7 @@ const uint32_t kLgAcSwingVHighest       = 0x881309D;
 const uint32_t kLgAcSwingVSwing         = 0x8813149;
 const uint32_t kLgAcSwingVAuto          = kLgAcSwingVSwing;
 const uint32_t kLgAcSwingVOff           = 0x881315A;
+const uint32_t kLgAcSwingVToggle        = 0x8810001;
```

Comment:
Would it make sense to order these codes by actual code (value)?
Looks like they already are and 0x8810001 just have to move to the top?

Reply:
Done.